actors/csharp/sdk/README.md

```diff
@@ -0,0 +1,152 @@
+# Quickstart: Actors (Dapr SDK) **In Development
+
+Let's take a look at the Dapr [Actors building block](https://docs.dapr.io/developing-applications/building-blocks/actors/actors-overview/). In this Quickstart, you will run a SmartDevice.Service microservice and a simple console client to demonstrate the stateful object patterns in Dapr Actors.  
+1. Using a SmartDevice.Service microservice, developers can host two SmartDectectorActor smoke alarm objects, and a third ControllerActor object that command and controls the smart devices.  
+2. Using a SmartDevice.Client console app, developers have a client app to interact with each actor, or the controller to perform actions in aggregate. 
+3. The SmartDevice.Interfaces contains the shared interfaces and data types used by both service and client apps
+
+> **Note:** This example leverages the Dapr client SDK.  
+
+
+### Step 1: Pre-requisites
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [.NET 7 SDK](https://dotnet.microsoft.com/download).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+### Step 2: Set up the environment
+
+Clone the [sample provided in the Quickstarts repo](https://github.com/dapr/quickstarts/tree/master/workflows).
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
+```
+
+### Step 3: Run the service app
+
+In a new terminal window, navigate to the `actors/csharp/sdk/service` directory and restore dependencies:
+
+```bash
+cd actors/csharp/sdk/service
+dotnet build
+```
+
+Run the `SmartDevice.Service`, which will start service itself and the Dapr sidecar:
+
+```bash
+dapr run --app-id actorservice --app-port 5001 --dapr-http-port 3500 --components-path ../../../resources -- dotnet run --urls=http://localhost:5001/
```

Comment:
```suggestion
dapr run --app-id actorservice --app-port 5001 --dapr-http-port 3500 --resources-path ../../../resources -- dotnet run --urls=http://localhost:5001/
```

Reply:
@paulyuk @marcduiker - One minor change whilst testing. I would suggest two things
1) Worth adding a register reminder timer on the controller that is called back after say 30s to turn off the alarms. This shows how to use a reminder
2) Work with @hhunter-ms to draw a diagram of the controller and the actors docs PR https://github.com/dapr/docs/pull/3309/files

Reply:
great suggestions @msfussell adding now. thanks for review